app/ante/validator_tx_fee.go

```diff
@@ -2,72 +2,94 @@ package ante
 
 import (
 	"math"
-	"strings"
 
 	errorsmod "cosmossdk.io/errors"
 	sdkmath "cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	parametertypes "github.com/elys-network/elys/v6/x/parameter/types"
-	oracletypes "github.com/ojo-network/ojo/x/oracle/types"
 )
 
-// CheckTxFeeWithValidatorMinGasPrices implements the default fee logic, where the minimum price per
-// unit of gas is fixed and set by each validator, can the tx priority is computed from the gas price.
-func CheckTxFeeWithValidatorMinGasPrices(ctx sdk.Context, tx sdk.Tx) (sdk.Coins, int64, error) {
-	feeTx, ok := tx.(sdk.FeeTx)
-	if !ok {
-		return nil, 0, errorsmod.Wrap(sdkerrors.ErrTxDecode, "Tx must be a FeeTx")
-	}
-
-	feeCoins := feeTx.GetFee()
-	gas := feeTx.GetGas()
+// GaslessAddrs contains only the governance module address.
+// Governance can grant feegrants to operational addresses (price feeders, liquidation bots, etc.)
+// When those addresses use feegrants, the fee payer becomes governance, enabling gasless transactions.
+var GaslessAddrs = []string{
```

Comment:
hard coding sender of tx makes tit difficult to change on the fly. Why not set it when chain starts by fetching params of respective module?
Only con is it might have to be done every block or we can do periodically or some other design